src/components/notifications/PagerDuty.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="mb-3">
+        <label for="pagerduty-integration-key" class="form-label">{{ $t("Integration Key") }}</label>
+        <HiddenInput id="pagerduty-integration-key" v-model="$parent.notification.pagerdutyIntegrationKey" :required="true" autocomplete="one-time-code"></HiddenInput>
+        <i18n-t tag="div" keypath="wayToGetPagerDutyKey" class="form-text">
+            <a
+                href="https://support.pagerduty.com/docs/services-and-integrations"
+                target="_blank"
```

Comment:
It might be worth adding `rel="noopener noreferrer"` to deal with security issues in older browsers


```suggestion
                rel="noopener noreferrer"
                target="_blank"
```

Reply:
All current versions of major browsers now automatically use the behaviour of `rel="noopener"` for any `target="_blank"` links.
And `noreferrer` is includes the functionality of `noopener`. 
So I'll leave that as is to keep inline with the rest of the codebase

Reply:
Fair enough. It was just for some of the legacy browsers that are probably floating around among some users. I agree with you, it is better to stick with the rest of the code base rather than change this given the number of users it could possibly affect.